CONTRIBUTING.rst

```diff
@@ -325,7 +325,42 @@ to avoid duplicate efforts.
     results in minor corrections at worst. We'll do our best to avoid larger
     problems in step 1.
 
+
+Notes on attrs
```

Comment:
This is really helpful, and exactly what I was looking for!

Reply:
Thanks! I'm going to update the contributing doc a bit more. It turns out that the new attr release from 9 days ago has few extra tricks, so I'll see if there is anything worth mentioning.

Reply:
@wilhadams I'm going to rerequest review since I've now updated the code and documentation to use a few nice features of attrs 20.1.0, which just got released.